src/components/Exchange/TradeHistoryV2.js

```diff
@@ -0,0 +1,180 @@
+import { useState } from "react";
+import Select from "react-select";
+import { useUserTradesData } from "../../Api";
+import Checkbox from "../Checkbox/Checkbox";
+import "./TradeHistory.css";
+import "./TradeHistoryV2.css";
```

Comment:
do we need separate style file?

Reply:
Sorry I was testing this and by mistake I left these files. I have deleted them form this PR